src/test/java/sorinfoca/Tests/Tema8/pages/DynamicButtonsSimplePage.java

```diff
@@ -0,0 +1,25 @@
+package sorinfoca.Tests.Tema8.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class DynamicButtonsSimplePage {
+
+    private WebDriver driver;
+
+    public DynamicButtonsSimplePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void get() {
```

Comment:
Please rename to goToPage() or something that suggests this action of navigating to url. In this way it will be easier to understand what this method do and not confusing with a getter method. Thanks!